js/cryptoLib/index.js

```diff
@@ -56,17 +56,32 @@ class Crypto {
                     let config = {
                         key: key
                     };
-                    const encrypt = crypto.encrypt(config);
+                    let encrypt;
+                    try {
+                        encrypt = crypto.encrypt(config);
+                    } catch (e) {
+                        log.send(logLevels.ERROR, 'Error decrypting : ' + e);
+                        if (fs.existsSync(`${this.dump}/${this.permanentIndexName}${searchConfig.TAR_LZ4_EXT}`)) {
```

Comment:
Could probably move the file name to a variable as this is being used in multiple places.